templates/bad_pr.j2

```diff
@@ -0,0 +1,9 @@
+Thanks @{{ submitter }}! This PR was evaluated as a bad PR for the following reasons:
+{% for reason in is_bad_pr_reason %}
+* {{ reason }}
+{% endfor %}
+
+Such PR can only be merged by human, ask a human to review this PR.
```

Comment:
We should point out how to contact an human :) Also maybe `s/human/core team member/` ?

Reply:
IRC is preferred